model/ftn/wmesmfmd.ftn

```diff
@@ -750,7 +751,10 @@
 !
 ! 1.e Set internal start/stop time from external start/stop time
 !
-      call ESMF_ClockGet(extClock, startTime=ttmp, rc=rc)
+
+!      call ESMF_ClockGet(extClock, startTime=ttmp, rc=rc)
+!      if (ESMF_LogFoundError(rc, PASSTHRU)) return
+      call ESMF_ClockGet(extClock, currTime=ttmp, rc=rc)
```

Comment:
To me this does not match the email thread from @ajhenrique  "Tim and I agreed that for now we should use the "if currTime > startTime" approach: in InitializeP1, get startTime, currTime, and stopTime from the external clock.  If currTime > startTime, then set WW3 stime from currTime, otherwise set WW3 stime from startTime."

Reply:
@JessicaMeixner-NOAA thanks again for noticing, these changes were made in a separate machine and not pushed to the repo. This will be completed shortly.